src/components/Proxy/proxyAutoReload.vue

```diff
@@ -0,0 +1,39 @@
+<script lang="ts" setup>
+import { ref, watchEffect } from 'vue';
+import { NCard, NSwitch, NTooltip } from 'naive-ui';
+
+import IconLabel from '@/components/IconLabel.vue';
+import useStore from '@/composables/useStore';
+
+const { proxyAutoReload } = useStore();
+
+const autoReloadLabel = ref('Auto reload disabled');
+const autoReloadDescription = ref(
+  'Opened tabs will automatically reload when associated proxy settings change.',
+);
+
+watchEffect(() => {
```

Comment:
Instead of using `ref` and `watchEffect` for `autoReloadLabel` and `autoReloadDescription` you can just use `computed`:
```
const autoReloadLabel = computed(() => proxyAutoReload.value ? 'Auto reload enabled' : 'Auto reload disabled');
...
```

Reply:
🧠 💨